src/Console/Application.php

```diff
@@ -286,10 +290,19 @@ public function buildContainer()
         };
 
         // VIOLATION RENDERER
-        $c['violation.renderer'] = function ($c) {
+        $c['violation.renderer.console'] = function ($c) {
             return new Renderer\ConsoleOutputRenderer(new ConsoleOutput(), $c['violation.message_helper']);
         };
 
+        $c['violation.renderer.html'] = function ($c) {
+            return new Renderer\HtmlOutputRenderer(
+                new ConsoleOutput(),
+                $c['violation.message_helper'],
+                new Filesystem(),
+                './output.html'
```

Comment:
You can update this PR by pushing more commits into your branch. No need for closing it. :-)